CHANGELOG.md

```diff
@@ -96,6 +96,7 @@ Here is an overview of all new **experimental** features:
 
 ### Fixes
 
+- **General**: Add AWS region to the AWS Config Cache key ([#6128](https://github.com/kedacore/keda/issues/6128))
 - **General**: Scalers cache uses a mutex to prevent concurrent actions ([#6273](https://github.com/kedacore/keda/issues/6273))
 - **AWS CloudWatch Scaler**: Properly parse multiple values of `dimensionName` & `dimensionValue` lists ([#6236](https://github.com/kedacore/keda/issues/6236))
```

Comment:
```suggestion
- **General**: Scalers cache uses a mutex to prevent concurrent actions ([#6273](https://github.com/kedacore/keda/issues/6273))
- **AWS Scalers**: Add AWS region to the AWS Config Cache key ([#6128](https://github.com/kedacore/keda/issues/6128))
- **AWS CloudWatch Scaler**: Properly parse multiple values of `dimensionName` & `dimensionValue` lists ([#6236](https://github.com/kedacore/keda/issues/6236))
```

Reply:
I'm not 100% sure, but won't that suggestion fail the CI build?

Reply:
f**k! yes, you're right!

Reply:
It should be alphabetically sorted. My point was to move the current like from **General** to *AWS Scalers*

Reply:
ah, ok, I changed the scaler name